stable-diffusion-webui-images-browser/javascript/images_history.js

```diff
@@ -0,0 +1,203 @@
+var images_history_click_image = function(){
+    console.log("in images_history_click_image !")
+    console.log(this.classList)
+    if (!this.classList.contains("transform")){        
+        var gallery = images_history_get_parent_by_class(this, "images_history_cantainor");
+        var buttons = gallery.querySelectorAll(".gallery-item");
+        var i = 0;
+        var hidden_list = [];
+        buttons.forEach(function(e){
+            if (e.style.display == "none"){
+                hidden_list.push(i);
+            }
+            i += 1;
+        })
+        if (hidden_list.length > 0){
+            setTimeout(images_history_hide_buttons, 10, hidden_list, gallery);
```

Comment:
I'm guessing this setTimeout is a hack, what you really need is async or a callback